documents/final-submission-document/final-submission-document.tex

```diff
@@ -102,19 +102,24 @@
 \vspace*{1ex}
 
 %Goal
-The goal of this thesis is to engineer a specialised and platform independent XMPP-Grid Broker that configures existing XMPP Controllers (XMPP server). The XMPP-Grid Broker should enable administrators to configure XMPP-Grids in a usable and productive way.
-
-The focus of the application should be set on portability, extensibility and the security aspects in a production environment.\\
+The goal of this thesis is to design and implement an XMPP-Grid Broker to configure existing Controllers (XMPP server), focussing on portability, extensibility and the security aspects in a production environment.
+The  Broker application should enable administrators to configure XMPP-Grids in a usable and productive way.\\
 \vspace*{1ex}
 
 %Result
-We propose the architecture of an XMPP-Grid Broker to configure XMPP Controllers and present an implementation based on the Angular5 framework.
+Our proposed architecture earmarks a purely client-side application that communicates typically through a regular reverse proxy with the XMPP Controller over WebSockets or HTTP streams (BOSH).
+Authentication is performed using mutual TLS authentication (client certificates) to conform to the IETF standard draft.
+The resulting application is implemented in TypeScript using the Angular5 framework.\\
+\vspace*{1ex}
 
-The resulting implementation enables administrators to create and configure communication Topics, apprehend the underlying hierarchy and manage permissions. Additionally, persistent items of Topics can be inspected and published.\\
+The resulting implementation enables administrators to create and configure communication Topics, apprehend the underlying hierarchy and manage permissions.
+Additionally, persistent items of Topics can be inspected and published.\\
```

Comment:
I just asked myself if we should write `Persistend Items` with capital letters here as "concept"?